controllers/attractor/lb-fe.go

```diff
@@ -69,15 +69,12 @@ func (l *LoadBalancer) getLbEnvVars(allEnv []corev1.EnvVar) []corev1.EnvVar {
 			Name:  "NSM_NSP_SERVICE",
 			Value: common.NSPServiceWithPort(l.trench),
 		},
-		{
```

Comment:
Is the outcome actually different?
I actually made this change on purpose. If using "valueFrom" in the manifests, there are some fields not specified but will be automatically filled by Kubernetes, and when comparing desired and current status, they will be different which triggers extra updates.

Reply:
I have check again, I remember the pods are using the pod name as unique identifier to request NSM connections

Reply:
Thanks for explanation